src/services/chat/prepopulated.js

```diff
@@ -41,15 +41,29 @@ module.exports = async (id, users) => {
             better_channel_member: [
               {
                 user: {
-                  ...ownUser.defaulValues
+                  updated_at: ownUser?.updated_at,
+                  is_anonymous: ownUser?.is_anonymous,
+                  name: ownUser?.username,
+                  image: ownUser?.profile_pic_path,
+                  last_active: ownUser?.last_active_at,
+                  username: ownUser?.username,
+                  id: ownUser?.user_id,
+                  created_at: ownUser?.created_at
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `name` property is being assigned the value of `ownUser?.username`, which might not be correct if `name` and `username` are different properties in the `ownUser` object. Please verify this assignment.
```diff
- name: ownUser?.username,
+ name: ownUser?.name,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->